.github/workflows/sonarcloud.yml

```
@@ -1,51 +1,78 @@
-
-      # # Install Python dependencies and run backend tests
-      # - name: Install Python dependencies and run backend tests
-      #   run: |
-      #     cd backend
-      #     pip install -r requirements.txt
-      #     pip install pytest coverage
-      #     pytest tests --cov=. --cov-report=xml --cov-report=term-missing  # Backend coverage report for Python
-      #   env:
-      #      DATABASE_URL: "sqlite:///:memory:"
-      # Run SonarCloud scan with coverage report
+          pytest backend/tests 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)